### PR TITLE
Using specific versions of plugin dependencies

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -29,9 +29,10 @@
     <engine name="cordova-ios" version="3.9.2" />
   </engines>
 
-  <dependency id="cordova-plugin-whitelist" />
-  <dependency id="cordova-plugin-device" />
-  <dependency id="phonegap-plugin-push" />
+  <!-- Specific versions of dependencies required by the Mobile SDK plugin -->
+  <dependency id="cordova-plugin-whitelist" url="https://github.com/apache/cordova-plugin-whitelist" commit="1.2.0" />
```
Comment (bhariharan) [inline plugin.xml]: ```commit``` could be any ```git-ref```. I've used tags here, but it could be SHA-1 too.

File: plugin.xml [hunk 2]
```diff
@@ -29,9 +29,10 @@
     <engine name="cordova-ios" version="3.9.2" />
   </engines>
 
-  <dependency id="cordova-plugin-whitelist" />
```
Comment (wmathurin) [inline plugin.xml]: whitelist is gone?

Comment (bhariharan) [inline plugin.xml]: No - they're all still there. It's just one line lower because of the comment line I added.

Comment (bhariharan) [inline plugin.xml]: I've also added URLs for all the plugins.

Comment (wmathurin) [inline plugin.xml]: Indeed, I really looked at the diff too quickly.

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**